File: cash.c
#include <stdio.h>
#include <math.h>
#include <cs50.h>


int main (void)
{
   double Valor;
   int centavos;

   do
   {
    Valor = get_double("Qual o valor inserido?\n");
    printf( "o troco e:%d\n");

   }
   while( Valor > 0);

   centavos = round(Valor * 100);

   int moedas = 0;

   moedas += centavos/25;
   centavos%= 25;

   moedas += centavos/10;
   centavos%= 10;

   moedas += centavos/5;
   centavos%= 5;

   moedas += centavos;

   printf("%d\n", moedas);

   return 0;
}

File: pset6/cash.py
from cs50 import get_float

while True:
  troco = get_float("Change: ")
  if troco < 1:
    break

cents = round (troco * 100)

moedas = 0

moedas += cents // 25
cents %= 25

moedas += cents // 10
cents %= 10

moedas += cents // 5
cents %= 5

moedas += cents

print(moedas)

